english/quick_draw/AI_Pictionary.txt

```diff
@@ -0,0 +1,5 @@
+It is a lot of fun playing pictionary with a bunch of friends. 
```

Comment:
Please make sure you name all of your files with the extension `.md`.  Otherwise, the files will not display correctly on GitHub.